png-afl/Cargo.toml

```diff
@@ -4,7 +4,8 @@ version = "0.2.0"
 authors = ["Sergey Davidoff <shnatsel@gmail.com>", "Paul Grandperrin <paul.grandperrin@gmail.com>"]
 
 [dependencies]
-afl = "0.4.0"
+afl = "0.4.3"
 png = {path = "../"}
 
-
+[features]
+fuzzing = []
```

Comment:
The feature would need to be placed in the `Cargo.toml` of the main library, and enabled by include the changing the reference line above to:

`png = { path = "../", features = ["fuzzing"] }

Reply:
Does your proposed way have advantages?

I tested this change locally by adding to png-afl/src/main.rs something like
```rust
fn main() {
    if cfg!(fuzzing) {
        panic!("fuzzing on!");
    }
    ...
}
``` 
rebuilt and invoked the binary directly

Reply:
The code in `src/main.rs` will be compiled with `cfg(fuzzing)` but not the artifacts that come from the included `png` library as far as I am aware.

Reply:
That makes sense. I'm still not successfully fuzzing then... :-)

Reply:
Features are local to the crate that declare them.

Reply:
Hmmm. It seems that `cargo afl build` sets the `--cfg fuzzing` itself.

Reply:
Because `cfg(fuzzing)` doesn't actualy select on a feature, the syntax for that would be `cfg(feature = "fuzzing")`.

Reply:
Yeah, it does. https://github.com/rust-fuzz/afl.rs/blob/e6fe251b42e09a9b512da7c0db11041c0a5653bf/src/bin/cargo-afl.rs#L156

Reply:
So, we've been fuzzing correctl all along, regardless of feature and RUSTFLAGS :+1:

Reply:
PR updated

Reply:
Huh. Missed your comments there :)

Reply:
Okay cool, dropping that stuff then

Reply:
PR updated with removed fuzzing feature